### DISCUSSION TITLE
Translate server

### DISCUSSION THREAD

**Discussion by aaaqueiroz** [GriefPrevention/GriefPrevention#938] [Q&A] [answered]
I need help, I translated the messages file and then he doesn't recognize the translation. How can I do to recognize?

**Comment by RoboMWM** [comment, marked as answer]
Must maintain valid yaml. Try wrapping your messages in double quotes. Use an online yaml parser/linter to doublecheck the syntax.

**Comment by aaaqueiroz** [comment]
Thank u
